tests/TestCase/Shell/BakeShellTest.php

```diff
@@ -87,7 +87,7 @@ public function testAllWithModelName()
             ->with('Comments');
 
         $this->Shell->connection = '';
-        $this->Shell->params = [];
+        $this->Shell->params = ['prefix' => 'account'];
```

Comment:
@andtxr well I'm pretty sure that I updated this line without the fix and it still works but nevermind